src/game/client/swarm/vgui/nb_mission_summary.cpp

```diff
@@ -139,6 +141,26 @@ void CNB_Mission_Summary::OnThink()
 		m_pMissionLabel->SetText(pMap->m_szMissionTitle);
 	}
 
+	int nSpeedrunTime = CAlienSwarm::GetSpeedrunTime();
+	wchar_t wszSRTime[ 128 ];
+
+	if ( nSpeedrunTime <= 0 )
+		V_snwprintf( wszSRTime, ARRAYSIZE( wszSRTime ), L"-" );
+	else
+		V_snwprintf( wszSRTime, ARRAYSIZE( wszSRTime ), g_pVGuiLocalize->FindSafe( "#nb_speedruntime_format" ), nSpeedrunTime / 60, nSpeedrunTime % 60 / 10, nSpeedrunTime % 10 );
```

Comment:
should use a format specifier like `%02d` instead of passing the seconds and ten-seconds separately